platform/core/src/services/CustomizationService/CustomizationService.ts

```diff
@@ -280,7 +280,7 @@ export default class CustomizationService extends PubSubService {
       return customization;
     }
     const parent = this.getCustomization(customizationType);
-    const result = parent ? Object.assign(Object.create(parent), customization) : customization;
+    const result = parent ? Object.assign({}, parent, customization) : customization;
```

Comment:
This isn't the same value here - is it fixing a bug?  If not, it means one can't assign a typed value as a default value, or have an object with hidden/non iterable values, which is useful sometimes.

Reply:
Yes, it's a bug fix. I'm not sure what the original intention was.